VMEncryption/main/DiskUtil.py

```diff
@@ -862,6 +862,12 @@ def get_azure_data_disk_controller_and_lun_numbers(self, dev_items_real_paths):
 
         return list_devices
 
+    def log_lsblk_output(self):
+        lsblk_command = 'lsblk -o NAME,TYPE,FSTYPE,LABEL,SIZE,RO,MOUNTPOINT'
```

Comment:
Does this give the type of setup like lvm/raid etc?

Reply:
Yes it does!